pkgs/applications/audio/lsp-plugins/default.nix

```diff
@@ -0,0 +1,144 @@
+{ stdenv, fetchFromGitHub, pkgconfig, makeWrapper
+, libsndfile, jack2Full
+, libGLU, libGL, lv2, cairo
+, ladspaH, php, expat }:
+
+stdenv.mkDerivation rec {
+  pname = "lsp-plugins";
+  version = "1.1.4";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "sadko4u";
+    repo = "${pname}";
+    rev = "${name}";
+    sha256 = "0vb8ax0w4d2a153wxrhkpi21fxsv7c24k57vhfgmm1lqwv6pbl69";
+  };
+
+  nativeBuildInputs = [ pkgconfig php expat ];
+  buildInputs = [ jack2Full libsndfile libGLU libGL lv2 cairo ladspaH makeWrapper ];
+
+  makeFlags = [
+    "BIN_PATH=$(out)/bin"
+    "LIB_PATH=$(out)/lib"
+    "DOC_PATH=$(out)/share/doc"
+  ];
+
+  patchPhase = ''
+    substituteInPlace Makefile --replace "/usr/lib" "$out/lib"
+    substituteInPlace ./include/container/jack/main.h --replace "/usr/lib" "$out/lib"
+    substituteInPlace ./include/container/vst/main.h --replace "/usr/lib" "$out/lib"
+  '';
+
+  buildPhase = "make all && make release";
+
```

Comment:
Does the package have tests?

Reply:
Yes, why?

Reply:
In that case please set ```doCheck = true;``` to run the tests.

Reply:
I've added that, but the build output doesn't indicate it's running any tests afaik.

Reply:
By default, when you set ```doCheck = true;``` it will run ```make check```. If that's not the name of the target, you can change that by setting ```checkTarget = "all-tests";``` or whatever the relevant make target is.

Reply:
I have it building tests now, but it looks like these tests are meant to be [ran manually](https://github.com/sadko4u/lsp-plugins/blob/master/README.txt#L235).

Shall I remove the tests all-together, or run them somehow?

Reply:
OK, it's fixed now!